src/unasync/__init__.py

```diff
@@ -156,8 +214,8 @@ def build_module(self, module, module_file, package):
         return outfile, copied
 
 
-def customize_build_py(rename_dir_from_to=("_async", "_sync")):
+def customize_build_py(rules=(_DEFAULT_RULE,)):
```

Comment:
This is not specific to this PR, but I'm not too happy that we have both `build_py` and `customize_build_py`. I think we can transform `customize_build_py` into `build_py.__call__`, and possibly only expose `build_py` and `Rule` as part of our API. What do you think?

Reply:
Agreed, I can make that change in this PR.

Reply:
Hmm so `build_py` is a class so we can't use `__call__` unfortunately. Maybe we can use `cmdclass_build_py(rules=[])` and make the `build_py` class a part of the private interface?